sap_business_document_processing/document_information_extraction_client/helpers.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-FileCopyrightText: 2020 2019-2020 SAP SE
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from .capabilities import HEADER_FIELD_NAMES_FOR_DOCUMENT_TYPES, LINE_ITEM_FIELDS_NAMES_FOR_DOCUMENT_TYPES
+from .constants import API_FIELD_CLIENT_ID, API_FIELD_DOCUMENT_TYPE, API_FIELD_ENRICHMENT, API_FIELD_TEMPLATE_ID, \
+    API_FIELD_EXTRACTED_HEADER_FIELDS, API_FIELD_EXTRACTED_LINE_ITEM_FIELDS, API_REQUEST_FIELD_EXTRACTED_FIELDS, \
+    API_REQUEST_FIELD_RECEIVED_DATE
+
+
+def create_document_options(client_id, document_type):
```

Comment:
Is this function used in more than 1 place?

Reply:
No, it is just used to simplify the code of the extract_information_from_documents function